src/components/Calendar/Event.js

```diff
@@ -0,0 +1,38 @@
+import React from 'react';
```

Comment:
Yes! good eye. Event.js is the component rendered in the EventListSideBar. it holds information about the sessions schedule for that day. I will give it a better name since it is too general.